packages/wdio-static-server-service/src/launcher.ts

```diff
@@ -1,43 +1,48 @@
-import path from 'path'
+import { join, resolve } from 'path'
 import { promisify } from 'util'
 
 import express from 'express'
 import fs from 'fs-extra'
 import morgan from 'morgan'
 import logger from '@wdio/logger'
+import { FolderOption, MiddleWareOption } from './types'
 
 const log = logger('@wdio/static-server-service')
 
 const DEFAULT_LOG_NAME = 'wdio-static-server-service.log'
 
 export default class StaticServerLauncher {
-    constructor ({ folders, port = 4567, middleware = [] }) {
+    folders: FolderOption[] | null
```

Comment:
Are you sure it can be null?

Reply:
Yes, as per the below line:

https://github.com/suniljaiswal01/webdriverio/blob/ts-static-server-service/packages/wdio-static-server-service/src/launcher.ts#L20